### PR TITLE
[DOCS-6239] Update advanced configuration options in Outlook docs

### PR DESCRIPTION
**Pull request by anxumalo**
Summary of changes:

- Adds `writeStreamBuffering` and `copyMoveWarningThreshold` (available since Outlook Integration 2.7)
- Removes `collapsible` (this is an old setting; equivalent to `showExplorer`) and `saml` (it's already integrated into **Connection settings**)
- Clarifies **Explore** tab tree view of the Alfresco sidebar

### CODE REVIEW

**Comment by anxumalo** [comment]
Hi @mhofmann-itnovum, as discussed, please review the fixes for the 2.7 & 2.8 docs. Many thanks

**Review by mhofmann-itnovum — DISMISSED** [review]
looks good

**Comment by anxumalo** [comment]
@mhofmann-itnovum I've now removed the parameters from the 2.7 page as well so it matches 2.8

File: microsoft-outlook/2.7/config/index.md [hunk 1]
```diff
@@ -1002,19 +1002,19 @@ Use this file to set up attributes and metadata settings.
...
     |`settingsCheckInterval`|Specifies the interval, in seconds, between checks to determine if the central settings have changed|`480`: 480 seconds is the default setting.|
     |`samlFallbackTimeout`|Sets a time, in seconds, for the Alfresco Outlook Client to wait before the SAML authentication window appears. This should only be required for slower networks to reduce the number of times that the authentication window reappears due to a delay from the server.<br><br>Added in Outlook Integration 2.4.7. Supported in versions 2.4.7 onwards and 2.6.|`7`: 7 seconds is the default setting.|
+    |`writeStreamBuffering`|Sets the `AllowWriteStreamBuffering` parameter of the HttpWebRequest.<br><br>**Note:** In a clustered Alfresco environment, you may encounter the error message _“This request requires buffering data to succeed”_ while uploading emails or files. Setting `writeStreamBuffering` to `true` will prevent this error from happening.<br><br>Added in Outlook Integration 2.7.|`true`: `AllowWriteStreamBuffering` is enabled.<br><br>`false`: `AllowWriteStreamBuffering` is disabled. This is the default setting.|
```
Comment (oliveralfresco2018) [inline microsoft-outlook/2.7/config/index.md:1005]: Are the underscores needed here?

File: microsoft-outlook/2.7/config/index.md [hunk 2]
```diff
@@ -1002,19 +1002,19 @@ Use this file to set up attributes and metadata settings.
...
     |`enableWFTab`|Controls the visibility of the Workflow tab in high resolution mode.<br><br>Added in Outlook Integration 2.6.|`true`: Workflow tab is visible.<br><br>`false`: Workflow tab is collapsed. This is the default setting.|
     |`enableWPF`|Enables/disables the use of a high resolution front-end for the Alfresco Outlook Client.<br><br>Added in Outlook Integration 2.6.|`true`: High resolution front-end is enabled. This is the default setting.<br><br>`false`: High resolution front-end is disabled.|
+    |`copyMoveWarningThreshold`|Sets the threshold for when a warning should be displayed, when a large amount of files is being copied/moved inside the repository with the copy/move & paste feature. Warns the user that copying `x` amount of files can take a long time depending on the server.<br><br>Added in Outlook Integration 2.7.|`100` is the default setting.|
```
Comment (oliveralfresco2018) [inline microsoft-outlook/2.7/config/index.md:1017]:
```suggestion
    |`copyMoveWarningThreshold`|Sets the threshold for when a warning should be displayed, when a large amount of files are being copied/moved inside the repository with the copy/move & paste feature. Warns the user that copying `x` amount of files can take a long time depending on the server.<br><br>Added in Outlook Integration 2.7.|`100` is the default setting.|
```

File: microsoft-outlook/latest/config/index.md [hunk 3]
```diff
@@ -1088,20 +1088,20 @@ Use this file to set up attributes and metadata settings.
...
     |`settingsCheckInterval`|Specifies the interval, in seconds, between checks to determine if the central settings have changed|`480`: 480 seconds is the default setting.|
     |`samlFallbackTimeout`|Sets a time, in seconds, for the Alfresco Outlook Client to wait before the SAML authentication window appears. This should only be required for slower networks to reduce the number of times that the authentication window reappears due to a delay from the server.<br><br>Added in Outlook Integration 2.4.7. Supported in versions 2.4.7 onwards and 2.6.|`7`: 7 seconds is the default setting.|
+    |`writeStreamBuffering`|Sets the `AllowWriteStreamBuffering` parameter of the HttpWebRequest.<br><br>**Note:** In a clustered Alfresco environment, you may encounter the error message _“This request requires buffering data to succeed”_ while uploading emails or files. Setting `writeStreamBuffering` to `true` will prevent this error from happening.<br><br>Added in Outlook Integration 2.7.|`true`: `AllowWriteStreamBuffering` is enabled.<br><br>`false`: `AllowWriteStreamBuffering` is disabled. This is the default setting.|
```
Comment (oliveralfresco2018) [inline microsoft-outlook/latest/config/index.md:1091]: Same comment as above.

File: microsoft-outlook/latest/config/index.md [hunk 4]
```diff
@@ -1088,20 +1088,20 @@ Use this file to set up attributes and metadata settings.
...
     |`enableWFTab`|Controls the visibility of the Workflow tab in high resolution mode.<br><br>Added in Outlook Integration 2.6.|`true`: Workflow tab is visible.<br><br>`false`: Workflow tab is collapsed. This is the default setting.|
     |`enableWPF`|Enables/disables the use of a high resolution front-end for the Alfresco Outlook Client.<br><br>Added in Outlook Integration 2.6.|`true`: High resolution front-end is enabled. This is the default setting.<br><br>`false`: High resolution front-end is disabled.|
     |`tokenAlterMode`|Used for QA and testing. Toggles the way a client is uniquely identified.|**Note:** Keep the default value: `false`.|
+    |`copyMoveWarningThreshold`|Sets the threshold for when a warning should be displayed, when a large amount of files is being copied/moved inside the repository with the copy/move & paste feature. Warns the user that copying `x` amount of files can take a long time depending on the server.<br><br>Added in Outlook Integration 2.7.|`100` is the default setting.|
```
Comment (oliveralfresco2018) [inline microsoft-outlook/latest/config/index.md:1104]: And again.

**Review by oliveralfresco2018 — CHANGES_REQUESTED** [review]
LGTM some minor comments.

File: microsoft-outlook/2.7/config/index.md (hunk 1 above)
Comment (anxumalo) [reply microsoft-outlook/2.7/config/index.md:1005]: Yes, they're useful for emphasising the error message.

File: microsoft-outlook/2.7/config/index.md (hunk 2 above)
Comment (anxumalo) [reply microsoft-outlook/2.7/config/index.md:1017]: I don't think this needs to change. The unit is "a large amount", singular, so no correction is needed even though there are multiple files.

File: microsoft-outlook/latest/config/index.md (hunk 4 above)
Comment (anxumalo) [reply microsoft-outlook/latest/config/index.md:1104]: See previous comment

File: microsoft-outlook/latest/config/index.md (hunk 3 above)
Comment (anxumalo) [reply microsoft-outlook/latest/config/index.md:1091]: See earlier comment - no change needed

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM

**Pull request merged**